subworkflows/local/variant_calling/call_snv_sentieon.nf

```diff
@@ -33,11 +35,17 @@ workflow CALL_SNV_SENTIEON {
             ch_vcf_idx = ch_vcf.join( ch_index )
 
             SENTIEON_DNAMODELAPPLY ( ch_vcf_idx, fasta, fai, ml_model )
-            ch_vcf      = SENTIEON_DNAMODELAPPLY.out.vcf
-            ch_index    = SENTIEON_DNAMODELAPPLY.out.vcf_index
             ch_versions = ch_versions.mix(SENTIEON_DNAMODELAPPLY.out.versions.first())
-        }
 
+            BCF_FILTER_ONE ( SENTIEON_DNAMODELAPPLY.out.vcf )
+
+            BCF_FILTER_TWO ( BCF_FILTER_ONE.out.vcf )
```

Comment:
If you decide on keeping the `BCF_FILTER_TOW` process inside the if statement you need to assign the output to this variable so that the correct vcf and index are joined below. 
```suggestion
            BCF_FILTER_TWO ( BCF_FILTER_ONE.out.vcf )
            ch_vcf = BCF_FILTER_TWO.out.vcf
```

Reply:
Alternatively, if you move out the `BCFTOOLS_FILTER_TWO` process together with `TABIX_BCFTOOLS` then you need to assign the vcf  output from the `BCFTOOLS_FILTER_ONE` to `ch_vcf`.